app/controllers/users_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 class UsersController < ApplicationController
   def index
-    @users = User.with_attached_avatar.order(:id).page(params[:page])
+    @users = User.order(:id).preload(avatar_attachment: :blob).page(params[:page])
```

Comment:
元々ここで `#with_attached_avatar` 使っているのはよさそうな気がするんですが、 `preload` で改善したんですね〜 🤔 
Followers / Followings で取得しているところが N+1 にならないかなーと心配でした。

Reply:
ご指摘の通り、フォローに関するユーザー一覧取得でのN+1問題が起きてました💦

`#preload`メソッドを使って取得するよう変更しました。

Reply:
[ここに書かれている](https://edgeapi.rubyonrails.org/classes/ActiveStorage/Attached/Model.html#method-i-has_one_attached) とおり、 t-koba さんが最初に書かれていた

> To avoid N+1 queries, you can include the attached blobs in your query like so:
```
User.with_attached_avatar
```
この書き方で N+1 が解決するはずなんですよね。
もしよかったら一回 `with_attached_avatar` に戻して、SQL の発行数などを確認してみてください〜！